src/data_access/Persistence.java

```diff
@@ -0,0 +1,6 @@
+package data_access;
+
+/* TODO: this needs to be modified to include the interfaces from the other
+ * use cases. */
+public interface Persistence {
```

Comment:
I'm not totally clear on the purpose of this Persistence interface, if some clarifying comments or Javadocs could be added that would be great.

Reply:
Thank you for adding Javadocs!